library/std/src/sync/once.rs

```diff
@@ -10,9 +10,15 @@ use crate::fmt;
 use crate::panic::{RefUnwindSafe, UnwindSafe};
 use crate::sys::sync as sys;
 
-/// A synchronization primitive which can be used to run a one-time global
-/// initialization. Useful for one-time initialization for FFI or related
-/// functionality. This type can only be constructed with [`Once::new()`].
+/// A low-level synchronization primitive for one-time global execution.
+///
+/// Once this was the only "execute once" synchronization provided by `std`.
```

Comment:
```suggestion
/// Previously, this was the only "execute once" synchronization provided by `std`.
```

to avoid confusion with the type name

Reply:
Aw, so much for sneaking that one by. :wink:

Reply:
`Once` upon a time... well, I suppose that is a bit cliche now.

Reply:
If it was intentional, so be it :laughing: But I think it would be mildly confusing — my brain parsed it wrong at first.